GRPC/courierServer.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+option go_package = "GRPC/courierProto";
+
+package courier;
+
+service CourierServer {
+  rpc CreateOrder(OrderCourierServer) returns (google.protobuf.Empty) {}
+  rpc GetDeliveryService(google.protobuf.Empty) returns (ServiceResponse) {}
```

Comment:
Do I understand correctly that this method returns a list of services? Then why is it named in singular word?

Reply:
https://cloud.google.com/apis/design/naming_convention#method_names

Reply:
I will fix)